config/settings.local.yaml.tpl

```diff
@@ -1,7 +1,7 @@
 #default:
 #  skip_cleanup: false
 #  tester: "someuser"                          # Optional: name of the user, who is running the tests, defaults to whoami/uid
-#  gateway_api: true                           # True, if Testsuite should test with Gateway API enabled (e.g. Full Kuadrant) or individual components (e.g. Authorino)
+#  standalone: false                           # True, if Testsuite should test only individual components (e.g. Authorino/limitador operators)
```

Comment:
I'd like to see all configurations defining a subset of tests that should run in one place (make). But this is for another PR if needed.

Reply:
Agree, will tackle it as part of a Follow-up PR